### PR TITLE
styling for events/:id

### PR DESCRIPTION
**Pull request by wickker**
able to display event card and pets coming with text on hover
added conditional to event date time display
able to use cards as checkboxes to add pets to event

### CODE REVIEW

File: app/views/events/show.html.erb [hunk 1]
```diff
@@ -1,90 +1,109 @@
...
+    <h2 class="mb-3 highlight">Who's Coming?</h2>
+    <div class="row card-deck">
+      <% if @event.pets.count != 0 %>
+        <% @event.pets.each do |pet| %>
+          <div class="col-sm-3 mr-0 mb-3">
+            <% if pet.species == "1"%>
+              <% species = "dog"%>
+            <% else %>
+              <% species = "cat"%>
+            <% end %>
```
Comment (dyanawu) [inline app/views/events/show.html.erb:53]:
```suggestion
```

File: app/views/events/show.html.erb [hunk 2]
```diff
@@ -1,90 +1,109 @@
...
+            <% owner = User.find(pet.user_id).username %>
+            <% popupText = "#{pet.name.capitalize} the #{species} with @#{owner}" %>
```
Comment (dyanawu) [inline app/views/events/show.html.erb:55]:
```suggestion
            <% popupText = "#{pet.name.capitalize} the #{pet.species.name} with @#{owner}" %>
```

File: app/models/event.rb [hunk 3]
```diff
@@ -41,15 +41,15 @@ def end_datetime
   end
 
   def start_date_str
-    self.start_date.getlocal.strftime("%d %B %Y")
+    self.start_time.getlocal.strftime("%d %B %Y")
```
Comment (dyanawu) [inline app/models/event.rb:44]: Oops thanks for catching these!

**Comment by wickker** [comment]
okie changed!

**Pull request merged**